src/ImageSharp.Web/Providers/PhysicalFileSystemProvider.cs

```diff
@@ -44,18 +43,12 @@ public class PhysicalFileSystemProvider : IImageProvider
             FormatUtilities formatUtilities)
         {
             Guard.NotNull(environment, nameof(environment));
-
-            // ContentRootPath is never null.
-            // https://github.com/dotnet/aspnetcore/blob/b89eba6c3cda331ee98063e3c4a04267ec540315/src/Hosting/Hosting/src/WebHostBuilder.cs#L262
```

Comment:
This comment exists to indicate why we do not do a null check for `environment.ContentRootPath`.

Reply:
I've updated this code to ensure it also works when `environment.WebRootPath` is null (and in that case use the same `wwwroot` default as [ASP.NET Core uses](https://github.com/dotnet/aspnetcore/blob/c85baf8db0c72ae8e68643029d514b2e737c9fae/src/Hosting/Hosting/src/Internal/HostingEnvironmentExtensions.cs#L31-L44)). This removes the need for any null checks here...